### PR TITLE
Added env vars for IPFS and ARWEAVE gateways.

### LINKED ISSUE #115: Document env vars for IPFS and ARWEAVE gateways.
Document env vars for IPFS and ARWEAVE gateways.

### PR DESCRIPTION
**Pull request by mariacarmina**
Fixes #115  .

Changes proposed in this PR:

- added env vars for IPFS and ARWEAVE gateways in README.md and Dockerfile

@alexcos20 mentioned [that](https://github.com/oceanprotocol/ocean-node/pull/94#discussion_r1393960952) once we have a native IPFS connection, we won't need the gateway, but for the moment I specified it.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -26,15 +26,38 @@ npm run build
 export HTTP_API_PORT=8000
 export PRIVATE_KEY=0x.....
 export RPCS="{ \"1\": \"https://rpc.eth.gateway.fm\", \"137\": \"https://polygon.meowrpc.com\", \"80001\": \"https://rpc-mumbai.maticvigil.com\" }"
+```
+
+For storing the file on IPFS or ARWEAVE, please export the following env variables;
```
Comment (jamiehewitt15) [inline README.md:31]: What files are we storing on IPFS and Arweave?

Comment (mariacarmina) [reply README.md:31]: If you have IPFS file or Arweave file object assigned to the DDO, you need to specify the gateways. It is a follow-up issues for https://github.com/oceanprotocol/ocean-node/pull/94.

Comment (jamiehewitt15) [reply README.md:31]: Ah ok, yeah makes sense

Comment (jamiehewitt15) [reply README.md:31]: But it's for downloading the files, rather than storing them?

Comment (mariacarmina) [reply README.md:31]: yes, I was referring to store in the DDO, I'll change into downloading. Thanks!

**Review by jamiehewitt15 — APPROVED** [review]
Looks good to me

**Review by alexcos20 — CHANGES_REQUESTED** [review]
let's do it properly:
 - add them to Config object
 - do checks at startup

**Comment by mariacarmina** [comment]
> let's do it properly:
> 
> * add them to Config object
> * do checks at startup

Ok, sure, thank you.

**Review by paulo-ocean — APPROVED** [review]
LGTM

**Review by alexcos20 — APPROVED** [review]

**Pull request merged**